### PR TITLE
First PR create  server

### PR DESCRIPTION
**Pull request by AntonioCorona-700**
Add the functionallty for hello path and for by path.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+   name := "human"
+   if len(parts) > 2 {
+       name = parts[2]
+   }
+   fmt.Fprintf(w, "Bye "+name)
+}
+
+func main() {
+   http.HandleFunc("/hello", handler)
+   http.HandleFunc("/bye", byeHandler)
```
Comment (rlara-sarmiento) [inline main.go:25]: You are very polite :)

**Review by rlara-sarmiento — APPROVED** [review]
LGTM

File: main.go [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+   "strings"
+)
+
+func handler(w http.ResponseWriter, r *http.Request) {
+   fmt.Println("I got a request!")
+   fmt.Fprintf(w, "Hi there :)")
+}
+func byeHandler(w http.ResponseWriter, r *http.Request) {
+   parts := strings.Split(r.URL.Path, "/")
+   name := "human"
```
Comment (rlara-sarmiento) [inline main.go:16]: Not sure if only humans will use this API

**Review by rlara-sarmiento — CHANGES_REQUESTED** [review]

File: main.go (hunk 2 above)
Comment (AntonioCorona-700) [reply main.go:16]: This specific option is only for humans.

Comment (rlara-sarmiento) [reply main.go:16]: You are right, sorry about that!

**Review by rlara-sarmiento — APPROVED** [review]
LGTM

**Pull request merged**